nucypher/network/protocols.py

```diff
@@ -33,7 +31,9 @@ def __init__(self, host, port) -> None:
 
     @classmethod
     def from_bytes(cls, url_string):
-        host_bytes, port_bytes = url_string.split(b":")
+        pos = url_string.find(b":")
```

Comment:
Question: would `url_string.split(":", 1)` achieve the same functionality? (asked as a python novice 😄 )

Reply:
Yes - I think `url_string.split(b":", 1)` will also work.

Reply:
Actually true, let's do that. It's more readable